src/main/common/printf.c

```diff
@@ -85,6 +123,7 @@ int tfp_format(void *putp, putcf putf, const char *fmt, va_list va)
             char lng = 0;
 #endif
             int w = 0;
+            int precision = 6; // Default float precision
```

Comment:
should precision be 7 (IEEE 754) ?

Reply:
@haslinghuis why not, maybe someone will need it